src/backend/dx11/src/device.rs

```diff
@@ -1519,10 +1538,15 @@ impl device::Device<Backend> for Device {
                     layers: 0..image.kind.num_layers(),
                 };
 
-                unordered_access_views.push(
-                    self.view_image_as_unordered_access(&view)
-                        .map_err(|_| device::BindError::WrongMemory)?,
-                );
+                let uav = self
+                    .view_image_as_unordered_access(&view)
+                    .map_err(|_| device::BindError::WrongMemory)?;
+
+                if let Some(ref mut name) = image.internal.debug_name {
+                    debug_name_with_suffix(&uav, name, &format!(" -- UAV Mip {}", mip));
```

Comment:
if we are going with `format!`, let's use the baseline `debug_name` for it. There is no point to pretend that `debug_name_with_suffix` helps here :)